srcpkgs/nawk/template

```diff
@@ -1,15 +1,17 @@
 # Template file for 'nawk'
 pkgname=nawk
-version=20121220
-revision=5
-create_wrksrc=yes
+version=20180823
+revision=1
+_commit=0f4e1ba922ccd45f7bd47701e923d98566ceb7c5
+wrksrc="awk-${_commit}"
 hostmakedepends="bison"
 short_desc="The one, true implementation of AWK"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="BSD"
```

Comment:
also please use SPDX identifier

Reply:
https://github.com/onetrueawk/awk/blob/master/LICENSE
I don't know precisely... what is this license?

Reply:
seems to be MIT but can't confirm

Reply:
https://spdx.org/licenses/StandardML-NJ.html `This license has been deprecated since 2.0rc2.`

Reply:
@Johnnynator looks like the right one... @maxice8 are you okay for SMLNJ ?